2024-09-09 21:41:48,789 - ERROR - Validation error at index 2: [{'type': 'uuid_parsing', 'loc': ('user_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `i` at 1', 'input': 'invalid-uuid', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `i` at 1'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-09 21:41:48,790 - ERROR - Validation error at index 3: [{'type': 'value_error', 'loc': ('email_address',), 'msg': 'value is not a valid email address: An email address must have an @-sign.', 'input': 'invalid-email', 'ctx': {'reason': 'An email address must have an @-sign.'}}, {'type': 'value_error', 'loc': ('join_date',), 'msg': 'Value error, Invalid date format: invalid-date', 'input': 'invalid-date', 'ctx': {'error': ValueError('Invalid date format: invalid-date')}, 'url': 'https://errors.pydantic.dev/2.8/v/value_error'}]
2024-09-09 21:49:45,902 - ERROR - Validation error at index 2: [{'type': 'uuid_parsing', 'loc': ('user_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `i` at 1', 'input': 'invalid-uuid', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `i` at 1'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-09 21:49:45,902 - ERROR - Validation error at index 2: [{'type': 'uuid_parsing', 'loc': ('user_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `i` at 1', 'input': 'invalid-uuid', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `i` at 1'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-09 21:49:45,903 - ERROR - Validation error at index 3: [{'type': 'value_error', 'loc': ('email_address',), 'msg': 'value is not a valid email address: An email address must have an @-sign.', 'input': 'invalid-email', 'ctx': {'reason': 'An email address must have an @-sign.'}}, {'type': 'value_error', 'loc': ('join_date',), 'msg': 'Value error, Invalid date format: invalid-date', 'input': 'invalid-date', 'ctx': {'error': ValueError('Invalid date format: invalid-date')}, 'url': 'https://errors.pydantic.dev/2.8/v/value_error'}]
2024-09-09 21:49:45,903 - ERROR - Validation error at index 3: [{'type': 'value_error', 'loc': ('email_address',), 'msg': 'value is not a valid email address: An email address must have an @-sign.', 'input': 'invalid-email', 'ctx': {'reason': 'An email address must have an @-sign.'}}, {'type': 'value_error', 'loc': ('join_date',), 'msg': 'Value error, Invalid date format: invalid-date', 'input': 'invalid-date', 'ctx': {'error': ValueError('Invalid date format: invalid-date')}, 'url': 'https://errors.pydantic.dev/2.8/v/value_error'}]
2024-09-09 21:51:06,006 - ERROR - Validation error at index 2: [{'type': 'uuid_parsing', 'loc': ('user_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `i` at 1', 'input': 'invalid-uuid', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `i` at 1'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-09 21:51:06,006 - ERROR - Validation error at index 2: [{'type': 'uuid_parsing', 'loc': ('user_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `i` at 1', 'input': 'invalid-uuid', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `i` at 1'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-09 21:51:06,006 - ERROR - Validation error at index 2: [{'type': 'uuid_parsing', 'loc': ('user_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `i` at 1', 'input': 'invalid-uuid', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `i` at 1'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-09 21:51:06,006 - ERROR - Validation error at index 2: [{'type': 'uuid_parsing', 'loc': ('user_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `i` at 1', 'input': 'invalid-uuid', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `i` at 1'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-09 21:51:06,006 - ERROR - Validation error at index 2: [{'type': 'uuid_parsing', 'loc': ('user_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `i` at 1', 'input': 'invalid-uuid', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `i` at 1'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-09 21:51:06,007 - ERROR - Validation error at index 3: [{'type': 'value_error', 'loc': ('email_address',), 'msg': 'value is not a valid email address: An email address must have an @-sign.', 'input': 'invalid-email', 'ctx': {'reason': 'An email address must have an @-sign.'}}, {'type': 'value_error', 'loc': ('join_date',), 'msg': 'Value error, Invalid date format: invalid-date', 'input': 'invalid-date', 'ctx': {'error': ValueError('Invalid date format: invalid-date')}, 'url': 'https://errors.pydantic.dev/2.8/v/value_error'}]
2024-09-09 21:51:06,007 - ERROR - Validation error at index 3: [{'type': 'value_error', 'loc': ('email_address',), 'msg': 'value is not a valid email address: An email address must have an @-sign.', 'input': 'invalid-email', 'ctx': {'reason': 'An email address must have an @-sign.'}}, {'type': 'value_error', 'loc': ('join_date',), 'msg': 'Value error, Invalid date format: invalid-date', 'input': 'invalid-date', 'ctx': {'error': ValueError('Invalid date format: invalid-date')}, 'url': 'https://errors.pydantic.dev/2.8/v/value_error'}]
2024-09-09 21:51:06,007 - ERROR - Validation error at index 3: [{'type': 'value_error', 'loc': ('email_address',), 'msg': 'value is not a valid email address: An email address must have an @-sign.', 'input': 'invalid-email', 'ctx': {'reason': 'An email address must have an @-sign.'}}, {'type': 'value_error', 'loc': ('join_date',), 'msg': 'Value error, Invalid date format: invalid-date', 'input': 'invalid-date', 'ctx': {'error': ValueError('Invalid date format: invalid-date')}, 'url': 'https://errors.pydantic.dev/2.8/v/value_error'}]
2024-09-09 21:51:06,007 - ERROR - Validation error at index 3: [{'type': 'value_error', 'loc': ('email_address',), 'msg': 'value is not a valid email address: An email address must have an @-sign.', 'input': 'invalid-email', 'ctx': {'reason': 'An email address must have an @-sign.'}}, {'type': 'value_error', 'loc': ('join_date',), 'msg': 'Value error, Invalid date format: invalid-date', 'input': 'invalid-date', 'ctx': {'error': ValueError('Invalid date format: invalid-date')}, 'url': 'https://errors.pydantic.dev/2.8/v/value_error'}]
2024-09-09 21:51:06,007 - ERROR - Validation error at index 3: [{'type': 'value_error', 'loc': ('email_address',), 'msg': 'value is not a valid email address: An email address must have an @-sign.', 'input': 'invalid-email', 'ctx': {'reason': 'An email address must have an @-sign.'}}, {'type': 'value_error', 'loc': ('join_date',), 'msg': 'Value error, Invalid date format: invalid-date', 'input': 'invalid-date', 'ctx': {'error': ValueError('Invalid date format: invalid-date')}, 'url': 'https://errors.pydantic.dev/2.8/v/value_error'}]
